src/tracer.js

```diff
@@ -25,6 +25,22 @@ class SignalFxTracer extends Tracer {
 
     this._scopeManager = new ScopeManager()
     this._scope = new Scope()
+
+    this._extraTags = {}
```

Comment:
I think this logic should be in Config: https://github.com/signalfx/signalfx-nodejs-tracing/blob/master/src/config.js#L62